packages/sdk/wallets/wallet-walletconnect/scripts/run-in-memory-client.ts

```diff
@@ -2,22 +2,30 @@ import { newKit } from '@celo/contractkit'
 import { WalletConnectWallet } from '../src'
 
 async function main() {
+  const name = `CLI DApp ${Math.random().toString().substring(12)}`
```

Comment:
jw: any reason for a random name if it's always using the use-contractkit app anyways?

Reply:
Just wanted to distinguish between test CLI clients vs. https://use-contractkit.vercel.app in the Valora screen. In Valora you see a list of connected sessions so instead of 

- use-contractkit
- use-contractkit

You'd see

- CLI DApp 123
- use-contractkit
- CLI DApp 456

Just helpful for debugging.

Reply:
cool makes sense, thanks for the clarification!